Source/WebCore/css/parser/CSSParserFastPaths.cpp

```diff
@@ -950,17 +933,39 @@ static RefPtr<CSSValue> parseColorWithAuto(StringView string, const CSSParserCon
 
 RefPtr<CSSValue> CSSParserFastPaths::maybeParseValue(CSSPropertyID propertyID, StringView string, const CSSParserContext& context)
 {
-    if (auto result = parseSimpleLengthValue(propertyID, string, context.mode))
-        return result;
-    if (propertyID == CSSPropertyDisplay)
+    switch (propertyID) {
+    case CSSPropertyDisplay:
         return parseDisplay(string);
-    if ((propertyID == CSSPropertyCaretColor || propertyID == CSSPropertyAccentColor) && isExposed(propertyID, &context.propertySettings))
-        return parseColorWithAuto(string, context);
+    case CSSPropertyOpacity:
```

Comment:
There are other properties that could benefit: fill-opacity/stroke-opacity/flood-opacity/stop-opacity (see `<alpha-value>` in CSSProperties.json)